recipes/boost/all/conanfile.py

```diff
@@ -604,7 +604,7 @@ def _build_bcp(self):
         with tools.vcvars(self.settings) if self._is_msvc else tools.no_op():
             with tools.chdir(folder):
                 command = "%s -j%s --abbreviate-paths toolset=%s" % (self._b2_exe, tools.cpu_count(), self._toolset)
-                if self.options.debug_level:
+                if "debug_level" in self.options:
                     command += " -d%d" % self.options.debug_level
```

Comment:
based on the docs I think this would be fine too https://boostorg.github.io/build/manual/develop/index.html#bbv2.overview.invocation.options

>-d0
Suppress all informational messages.
-d N
Enable cumulative debugging levels from 1 to n. Values are: +

It's unclear if a space is required

```suggestion
                command += " -d%d" % self.options.get_safe("debug_level", 0)
```

Reply:
Thanks, @prince-chrismc.

For least surprise, should we default to the same value as `b2` does itself, which I understand from that linked document to be level `1`?

I'm pretty sure that the space is not required.

Reply:
I picked the default for the recipe which was already accepted 🤷 we'll see what the CCI team and community say

Reply:
from my experience, space isn't required
using b2 default as recipe default matches the expectation for the most of users